Resources/public/js/views/ez-contenteditview.js

```diff
@@ -292,7 +289,13 @@ YUI.add('ez-contenteditview', function (Y) {
              */
             formView: {
                 valueFn: function () {
-                    return new Y.eZ.ContentEditFormView();
+                    return new Y.eZ.ContentEditFormView({
+                        config: this.get('config'),
+                        contentType: this.get('contentType'),
+                        content: this.get('content'),
+                        version: this.get('version')
+
```

Comment:
nitpick: useless empty line (and while at it you can add a coma at the end of previous line)